examples/headsail-bsp/src/mmap.rs

```diff
@@ -1,2 +1,5 @@
 pub(crate) const UART0_ADDR: usize = 0xFFF00000;
-pub(crate) const TIMER0_ADDR: usize = 0x5_0000;
\ No newline at end of file
+pub(crate) const TIMER0_ADDR: usize = 0x5_0000;
+pub(crate) const TIMER1_ADDR: usize = 0x5_0010;
+pub(crate) const TIMER2_ADDR: usize = 0x5_0020;
+pub(crate) const TIMER3_ADDR: usize = 0x5_0030;
```

Comment:
Are these memory maps correct for both VP and ASIC?

Reply:
Yes, those are the addresses provided in the GitLab pages relevant documentation.